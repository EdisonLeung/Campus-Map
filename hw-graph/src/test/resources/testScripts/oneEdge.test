# Test adding one edge from one node to another

# Create graph
CreateGraph oneEdge

# Add 2 nodes
AddNode oneEdge a
AddNode oneEdge b

# Add edge
AddEdge oneEdge a b AB

# Print nodes and children
ListNodes oneEdge
ListChildren oneEdge a
ListChildren oneEdge b