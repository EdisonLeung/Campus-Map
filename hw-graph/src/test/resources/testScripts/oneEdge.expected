# Test adding one edge from one node to another

# Create graph
created graph oneEdge

# Add 2 nodes
added node a to oneEdge
added node b to oneEdge

# Add edge
added edge AB from a to b in oneEdge

# Print nodes and children
oneEdge contains: a b
the children of a in oneEdge are: b(AB)
the children of b in oneEdge are: